ibis/expr/datatypes/tests/test_core.py

```diff
@@ -116,16 +146,12 @@ class BarStruct:
     m: dt.Date
     n: dt.Time
     o: dt.Timestamp
-    oa: dt.Timestamp['UTC']  # noqa: F821, UP037
```

Comment:
Are we testing these elsewhere?

Reply:
No, these are not supported anymore since timestamp is not generic over timezone.